packages/x-date-pickers-pro/src/MultiInputDateRangeField/MultiInputDateRangeField.tsx

```diff
@@ -29,6 +29,13 @@ const MultiInputDateRangeFieldSeparator = styled(
   },
 )({});
 
+const MultiInputDateRangeFieldTextField = styled(MuiTextField, {
+  name: 'MuiMultiInputDateRangeField',
+  slot: 'TextField',
+  overridesResolver: (props, styles) => styles.textField,
```

Comment:
Am I missing something or this does not make much sense?
I don't see an option to override this class slot. 🤔 
If we are fine with the `fullWidth` approach, this styled component can probably be removed. 👌

Reply:
Indeed, with the prop, this styled component has no use :+1: